joint_name:
  - 'Neck'
  - 'Head'
  - 'ShoulderR'
  - 'ArmUpperR'
  - 'ArmLowerR'
  - 'ShoulderL'
  - 'ArmUpperL'
  - 'ArmLowerL'
  - 'PelvYR'
  - 'PelvR'
  - 'LegUpperR'
  - 'LegLowerR'
  - 'AnkleR'
  - 'FootR'
  - 'PelvYL'
  - 'PelvL'
  - 'LegUpperL'
  - 'LegLowerL'
  - 'AnkleL'
  - 'FootL'

sensor_name:
  - 'NeckS'
  - 'HeadS'
  - 'ShoulderRS'
  - 'ArmUpperRS'
  - 'ArmLowerRS'
  - 'ShoulderLS'
  - 'ArmUpperLS'
  - 'ArmLowerLS'
  - 'PelvYRS'
  - 'PelvRS'
  - 'LegUpperRS'
  - 'LegLowerRS'
  - 'AnkleRS'
  - 'FootRS'
  - 'PelvYLS'
  - 'PelvLS'
  - 'LegUpperLS'
  - 'LegLowerLS'
  - 'AnkleLS'
  - 'FootLS'

velocity_limit: 24.5324 # rad/s
position_min: -3.14159 # rad
position_max: 3.14159 # rad